airflow_provider_fivetran/operators/fivetran.py

```diff
@@ -1,9 +1,29 @@
-from airflow.models import BaseOperator
+import logging
+import json
+import time
+
+import requests
+
+from airflow.models import BaseOperator, BaseOperatorLink
 from airflow.utils.decorators import apply_defaults
-from typing import Optional
+from typing import Any, Dict, List, Optional, Union
+
 
 from airflow_provider_fivetran.hooks.fivetran import FivetranHook
 
+log = logging.getLogger(__name__)
```

Comment:
I think this was accidentally re-added. Can you confirm you are using `log`?

Reply:
I'm not using log I'll remove it